project/Build.scala

```diff
@@ -35,8 +35,15 @@ object SlickBuild extends Build {
     val reactiveStreamsVersion = "1.0.2"
     val reactiveStreams = "org.reactivestreams" % "reactive-streams" % reactiveStreamsVersion
     val reactiveStreamsTCK = "org.reactivestreams" % "reactive-streams-tck" % reactiveStreamsVersion
-    val hikariCP = "com.zaxxer" % "HikariCP" % "2.7.4"
-    val mainDependencies = Seq(slf4j, typesafeConfig, reactiveStreams)
+    val hikariCP = "com.zaxxer" % "HikariCP" % "2.7.8"
+    val jaxbVersion = "2.3.0"
+    val jaxb = Seq(
+      "javax.xml.bind" % "jaxb-api" % jaxbVersion % "provided",
+      "com.sun.xml.bind" % "jaxb-core" % jaxbVersion,
+      "com.sun.xml.bind" % "jaxb-impl" % jaxbVersion
```

Comment:
I think these jaxb dependencies are no longer needed now that #2008 has been merged